exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -205,10 +225,10 @@ drill.exec: {
         // Deprecated for managed xsort; used only by legacy xsort
         threshold: 40000,
         // File system to use. Local file system by default.
-        fs: "file:///"
+        fs: ${drill.exec.spill.fs},
```

Comment:
Suggestion: explain what we are doing:

In Drill versions prior to 1.11, only sort spilled. Starting in 1.11, other operators spill.
Spilling is now configured in the spill group above. For backward compatibility, existing configuration files will still configure spilling here. Please move spill configuration to the new group above.